HISTORY.md

```diff
@@ -1,5 +1,56 @@
 ### mlpack ?.?.?
 ###### ????-??-??
+  * Migrate from boost tests to Catch2 framework (#2523), (#2584).
+
+  * Update NumFOCUS team voting responsibilities (#3071).
+
+  * Add armadillo bump version policy (#3048).
+
+  * Bump minimum armadillo version from 8.400 to 9.800 (#3043).
```

Comment:
I would move `#3048` into the `Bump minimum armadillo version from 8.400 to 9.800 (#3043).`, similar reason as mentioned above.